eagle-stores/src/context/auth/auth-context.js

```diff
@@ -0,0 +1,43 @@
+import { createContext, useContext, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+
+const AuthContext = createContext();
+
+const AuthProvider = ({ children }) => {
+  const navigate = useNavigate();
+  const localStorageToken = JSON.parse(localStorage.getItem('login'));
+  const [token, setToken] = useState(localStorageToken?.token);
+  const [user, setUser] = useState(null);
+
+  const loginHandler = async (email, password) => {
+    try {
+      const resp = await axios.post('/api/auth/login', {
```

Comment:
better to make a services.js file and defined these req as a function there and then use those calling functions here
i.e const LoginService=async()=>{
 return axios.post("api")
}

Reply:
Thanks for the suggestion i will definitely make these changes